quiz/web/tailwind.config.cjs

```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+    content: ["./src/**/*.{html,js,jsx,ts,tsx}"],
```

Comment:
include `index.html` otherwise tailwind will not parse styles in `index.html` unlikely that it will be necessary, but in the future you'll spend eternity trying to figure out why tailwind doesn't work for this file.